hrjobcontract/js/test/services/utils_test.js

```diff
@@ -0,0 +1,47 @@
+define([
+    'common/angular',
+    'common/angularMocks',
+    'job-contract/app'
+], function (angular, Mock) {
+    'use strict';
+
+    describe('UtilsService', function () {
+        var UtilsService;
+        var API;
+
+        beforeEach(module('hrjc'));
+        beforeEach(inject(function (_UtilsService_, _API_) {
+            UtilsService = _UtilsService_;
+            API = _API_;
+        }));
+
+        beforeEach(function () {
+            spyOn(API, 'resource').and.callFake(function () { return { get: function () {} } })
+            UtilsService.getAbsenceType();
+            UtilsService.getHoursLocation();
+            UtilsService.getPayScaleGrade();
+        });
+
+        describe('getAbsenceType', function () {
+            it('Test getAbsenceType', function () {
```

Comment:
this is not phpunit where there is no way to decently give a description to the test :D give a meaningful one

(remember to be imperative, as in "does this...". instead of "should do this...")